dipy/viz/horizon/app.py

```diff
@@ -443,7 +442,8 @@ def build_show(self, scene):
             img_count = 0
             synchronize_slices = check_img_shapes(self.images)
             for img in self.images:
-                data, affine = img
+                data, affine, fname = img
+                fname = fname.split("/")[-1]
```

Comment:
splitting the file path and extracting the name

Reply:
Are you sure it works on windows? I will be surprised because they usually use `\\`.  I recommend you to use `os.path.basename`. You can also look at the `Pathlib` python module.